core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1605,6 +1608,8 @@ object SnappySession extends Logging {
     CacheBuilder.newBuilder().maximumSize(300).build(loader)
   }
 
+  def getPlanCache = planCache
```

Comment:
Doesn't matter, with proper codestyle settings this will flag with red color in the editor.